src/test/objectstore/Fragmentation_simulator.cc

```diff
@@ -93,7 +96,10 @@ int FragmentationSimulator::begin_simulation_with_generators() {
       return r;
```

Comment:
https://github.com/ceph/ceph/pull/52305/files#diff-6f70865ed07b8c56aec3a08c6657b5f277d42efb2ac329638868788220099ba7L85-L92
I have 2 comments about this loop:
- when there is more then 1 genetator, we create same collection multiple times.
- the overall construction does not allow for multiplexing from multiple generators,

Reply:
I've changed to using random distribution to pick the generator, also begin simulation now takes in `iterations` which just tells it how many time to ask for new workloads